### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by aedrax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 -----
+![SCUBA](https://user-images.githubusercontent.com/5616068/195471179-ef1b04e7-0d45-4f81-9460-e1af8a150199.png)
 
-Simple Container-Utilizing Build Apparatus
+**SCUBA** is the **S**imple **C**ontainer-**U**tilizing **B**uild **A**pparatus
 
 Scuba makes it easier to use Docker containers in everyday development. It is
-intended to be used by developers with `make` or `scons`-based build systems,
-where the entire build environment is encapsulated in a Docker container.
+intended to allow a developer to commit an environment setup where the entire
```
Comment (JonathonReinhart) [inline README.md:8]: I like changing this to make it more generic. Can we drop the "intended to" and make it just "allows"? I.e. "It allows a developer..."

**Review by JonathonReinhart — CHANGES_REQUESTED** [review]
Thanks Paul! I have a few minor requests if you wouldn't mind.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
 SCUBA  [![Build Status](https://github.com/JonathonReinhart/scuba/actions/workflows/build-test.yml/badge.svg)](https://github.com/JonathonReinhart/scuba/actions/workflows/build-test.yml) [![codecov.io](https://codecov.io/github/JonathonReinhart/scuba/coverage.svg?branch=master)](https://codecov.io/github/JonathonReinhart/scuba?branch=master) [![PyPI](https://img.shields.io/pypi/v/scuba.svg)](https://pypi.python.org/pypi/scuba) [![Docs](https://readthedocs.org/projects/scuba/badge/?version=latest)](https://scuba.readthedocs.io/)
 -----
+![SCUBA](https://user-images.githubusercontent.com/5616068/195471179-ef1b04e7-0d45-4f81-9460-e1af8a150199.png)
```
[the remark this answers is not in the document]
Comment (aedrax) [reply README.md:3]: Thanks! I'll get that added in! :)

File: README.md [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
-Simple Container-Utilizing Build Apparatus
+**SCUBA** is the **S**imple **C**ontainer-**U**tilizing **B**uild **A**pparatus
 
 Scuba makes it easier to use Docker containers in everyday development. It is
-intended to be used by developers with `make` or `scons`-based build systems,
-where the entire build environment is encapsulated in a Docker container.
+intended to allow a developer to commit an environment setup where the entire
+build environment is encapsulated in a Docker container.
+
+![demo](https://user-images.githubusercontent.com/5616068/195481514-a1bdf66b-531b-42b1-b149-efe7b83108b8.gif)
```
[the remark this answers is not in the document]
Comment (aedrax) [reply README.md:11]: lol, will do!

File: README.md (hunk 1 above)
Comment (aedrax) [reply README.md:8]: Consider it done

**Comment by aedrax** [comment]
All the requested changes are pushed :)

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (JonathonReinhart) [reply README.md]: Looks great: https://scuba--193.org.readthedocs.build/

**Review by JonathonReinhart — APPROVED** [review]

**Pull request merged**